recipes/kit4dl/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "kit4dl" %}
+{% set version = "2023.08b0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  path: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kit4dl-{{ version }}.tar.gz
+  sha256: 7ab8579afdf042b32559b9cc8eac72ed34d3dbd1fba87e0dfa09353cf7d65ce4
+
+build:
```

Comment:
It looks like this package has a command line entry point, so you'll want to add the following under `build`.

```
entry_points:
    - kit4dl = kit4dl.cli.app:run
 ```

And then under `tests` > `commands`, test the cli:

```
commands:
    - pip check
    - kit4dl --help
requires:
    - pip
 ```